app/assets/javascripts/components/HomeLogin.jsx

```diff
@@ -0,0 +1,116 @@
+class HomeLogin extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      error: '',
+      hasError: false
+    }
+  };
+
+  _getToken() {
+    const token = document.getElementsByName("csrf-token")[0].getAttribute("content");
+    return token;
+  };
+
+  _handleKeydown(k) {
+    if (k.keyCode == 13) {
+      this._handleLogin();
+    }
+  }
+
+  handleChange(event) {
+    this.setState({ [$(event.target).attr("name")] : $(event.target).val() });
+  }
+
+  _handleLogin() {
```

Comment:
Change all of the 
"_handleSomething" functions to be class properties that are arrow functions. This way we dont have to use arrow functions when we pass them into other components.
like:

_handleLogin = () => {
 stuff
}